### DISCUSSION TITLE
actions/cache@v1 is slow to restore

### DISCUSSION THREAD

**Discussion by davetapley** [community/community#25087] [Actions]
<p>We’re trialing Actions as a replacement for Semaphore CI.</p>
<p>Semaphore CI have a <a href="https://docs.semaphoreci.com/article/149-caching" rel="nofollow noopener">cache</a> similar to  actions/cache@v1 but it only takes <strong>six seconds</strong> to restore our node_modules one Semaphore, compared to  <strong>twenty eight seconds</strong> on GitHub actions.</p>
<p>Is this a known deficiency?</p>

**Comment by weide-zhou** [comment]
<p>Hi <a class="mention" href="/dukedave">@dukedave</a> ,</p>
<p>Much appreciate your feedback! We’re always working to improve GitHub, and we consider every suggestion we received. I have submitted a query internally based on your ticket, will update once there is a response! Thanks.</p>

**Comment by yaananth** [comment, marked as answer]
<p><a class="mention" href="/dukedave">@dukedave</a> Thank you the feedback.</p>
<p><a href="https://github.com/actions/cache/issues" rel="nofollow noopener">https://github.com/actions/cache/issues</a> is the right place for it.</p>
<p>It would also help if you have particular examples showing the performance differences. That would help us to pin point where the lag is.</p>
<p>Please do take some time and create an issue over there with examples showing the difference, this would help us.</p>

**Comment by davetapley** [comment]
<p>Thanks, done!</p>
<p><a href="https://github.com/actions/cache/issues/139" rel="nofollow noopener">https://github.com/actions/cache/issues/139</a></p>
